copier/main.py

```diff
@@ -277,7 +282,15 @@ def _execute_tasks(self, tasks: Sequence[Task]) -> None:
                 )
             if self.pretend:
                 continue
-            with local.cwd(self.subproject.local_abspath), local.env(**task.extra_env):
+
+            working_directory = (
+                    # We can't use _render_path here, as that function has special handling for files in the template
+                    self.subproject.local_abspath / Path(self._render_string(str(task.working_directory), extra_context))
```

Comment:
I don't think that's needed, but it could make the code a bit clearer.
The current solution also allows using absolute paths:
```python
>>> from pathlib import Path
>>> p = Path("/user/test")
>>> p
PosixPath('/user/test')
>>> p / "/user/other"
PosixPath('/user/other')
```

Reply:
Ah I didn't know that, thanks.